Backend/NoCode.FlowerShop.Infrastructure/Persistence/Configurations/FlowerConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using NoCode.FlowerShop.Domain;
+
+namespace NoCode.FlowerShop.Infrastructure.Persistence.Configurations;
+
+public class FlowerConfiguration : IEntityTypeConfiguration<Flower>
+{
+    public void Configure(EntityTypeBuilder<Flower> builder)
+    {
+        ConfigureCustomersTable(builder);
+    }
+
+    private static void ConfigureCustomersTable(EntityTypeBuilder<Flower> builder)
```

Comment:
I think there is a name mismatch here. It should be called `ConfigureFlowersTable`